doppler/resource_sync_types.go

```diff
@@ -187,3 +187,29 @@ func resourceSyncTerraformCloud() *schema.Resource {
 	}
 	return builder.Build()
 }
+
+func resourceSyncFlyio() *schema.Resource {
+	builder := ResourceSyncBuilder{
+		DataSchema: map[string]*schema.Schema{
+			"app_id": {
+				Description: "The app ID ",
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+			},
+			"restart_machines": {
+				Description: "The Terraform Cloud variable set ID to sync to",
```

Comment:
This description seems wrong. Copypasta?

Reply:
Good catch 👍